app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/TrackerBlockingVpnService.kt

```diff
@@ -217,7 +217,12 @@ class TrackerBlockingVpnService : VpnService(), CoroutineScope by MainScope() {
             // See https://developer.android.com/reference/android/net/VpnService.Builder#allowFamily(int) for more info as to why
             allowFamily(AF_INET6)
 
-            VpnRoutes.includedRoutes.forEach { addRoute(it.address, it.maskWidth) }
+            if (appBuildConfig.isPerformanceTest) {
```

Comment:
I am wondering if we should hold off on merging this change until we have some UI-based test that smoke-checks that we block trackers (when on a non-performance test build).